src/jdk.accessibility/windows/native/jabswitch/jabswitch.cpp

```diff
@@ -323,15 +323,15 @@ void printVersion() {
 
 int regEnable() {
     HKEY hKey;
-    DWORD retval = -1;
+ // DWORD retval = -1;
```

Comment:
I don't think we need to comment out both retval and freeData. It is either remnants of the code that was long gone or (more likely) were just a copy of another template method that actually uses these variables. If variables are not used within the function i would just delete them.

Reply:
Will do, thanks

Reply:
I'll remove retval in a moment